docs/utilities/batch.md

```diff
@@ -1,460 +1,322 @@
+
+With this utility, batch records are processed individually – only messages that failed to be processed 
+return to the queue or stream for a further retry. You simply build a `BatchProcessor` in your handler,
+and return its response from the handler's `processMessage` implementation. Exceptions are handled 
+internally and an appropriate partial response for the message source is returned to Lambda for you.
 
 !!! warning
-    While this utility lowers the chance of processing messages more than once, it is not guaranteed. We recommend implementing processing logic in an idempotent manner wherever possible.
+    While this utility lowers the chance of processing messages more than once, but it is not guaranteed. 
+    We recommend implementing processing logic in an idempotent manner wherever possible, for instance,
+    by taking advantage of [the idempotency module](idempotency.md).
     More details on how Lambda works with SQS can be found in the [AWS documentation](https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html)
 
 ## Install
 
-Depending on your version of Java (either Java 1.8 or 11+), the configuration slightly changes.
-
-=== "Maven Java 11+"
-
-    ```xml hl_lines="3-7 16 18 24-27""
-    <dependencies>
-        ...
-        <dependency>
-            <groupId>software.amazon.lambda</groupId>
-            <artifactId>powertools-sqs</artifactId>
-            <version>{{ powertools.version }}</version>
-        </dependency>
-        ...
-    </dependencies>
-    ...
-    <!-- configure the aspectj-maven-plugin to compile-time weave (CTW) the aws-lambda-powertools-java aspects into your project -->
-    <build>
-        <plugins>
-            ...
-            <plugin>
-                 <groupId>dev.aspectj</groupId>
-                 <artifactId>aspectj-maven-plugin</artifactId>
-                 <version>1.13.1</version>
-                 <configuration>
-                     <source>11</source> <!-- or higher -->
-                     <target>11</target> <!-- or higher -->
-                     <complianceLevel>11</complianceLevel> <!-- or higher -->
-                     <aspectLibraries>
-                         <aspectLibrary>
-                             <groupId>software.amazon.lambda</groupId>
-                             <artifactId>powertools-sqs</artifactId>
-                         </aspectLibrary>
-                     </aspectLibraries>
-                 </configuration>
-                 <executions>
-                     <execution>
-                         <goals>
-                             <goal>compile</goal>
-                         </goals>
-                     </execution>
-                 </executions>
-            </plugin>
-            ...
-        </plugins>
-    </build>
-    ```
+We simply add `powertools-batch` to our build dependencies. Note - if you are using other Powertools
+modules that require code-weaving, you will need to configure that also. Batch does not.
```

Comment:
Batch does not?

Reply:
unclear, we probably don't need what to do for other modules, just batch

Reply:
I rephrased this a bit. Batch doesn't do any weaving.